assignments/objects.js

```diff
@@ -10,55 +10,128 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
-
-// Write your intern objects here:
+// const example = {
+//   "id": 0,
+//   "name": "Example",
+//   "email": "examples@you.edu",
+//   "gender": "F"
+// }
+
+// // Write your intern objects here:
+// const intern1 = {
+//   "id": 1,
+//   "name": "Mitzi",
+//   "email": "mmelloy0@psu.edu",
+//   "gender": "F",
+// }
+
+// const intern2 = {
+//   "id": 2,
+//   "name": "Kennan",
+//   "email": "kdiben1@tinypic.com",
+//   "gender": "M",
+// }
+
+// const intern3 = {
+//   "id": 3,
+//   "name": "Keven",
+//   "email": "kmummery2@wikimedia.org",
+//   "gender": "M",
+// }
+
+// const intern4 = {
+//   "id": 4,
+//   "name": "Gannie",
+//   "email": "gmartinson3@illinois.edu",
+//   "gender": "M",
+// }
+
+// const intern5 = {
+//   "id": 5,
+//   "name": "Antonietta",
+//   "email": "adaine5@samsung.com",
+//   "gender": "F",
+// }
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+// console.log(intern1.name);
 
-// Kennan's ID
+// // Kennan's ID
+// console.log(intern2.id);
 
-// Keven's email
+// // Keven's email
+// console.log(intern3.email);
 
-// Gannie's name
+// // Gannie's name
+// console.log(intern4.name);
 
-// Antonietta's Gender
+// // Antonietta's Gender
+// console.log(intern1.gender);
```

Comment:
intern5 here, not that it really matters lolol

Reply:
Oops. A simple mistake like that could really mess something up. I'll be sure to double, triple check my work next time.